### ISSUE TITLE
How to "smoothen" the experience

### ISSUE THREAD

**Issue by Kampii** [JeremyHeleine/Photo-Sphere-Viewer#17]
Here (https://theta360.com/spheres/samples/532f020e-4569-11e5-a824-064087ff3472-1) is an example of the user experience I would like to achieve. When user drags the image it sort of sticks to the grab point. What settings do I change in order to achieve this?

**Comment by JeremyHeleine** [comment]
Hi,

You can choose the speed of the user's movements with the options `long_offset` and `lat_offset` (you can see an example of use in the `example1.js` file). Is it what you were searching for?

**Comment by Kampii** [comment]
Thanks, let me check.

In the "theta360" site when the user grabs (clicks) some point on the image to drag it, that point follows the mouse more or less. Or like when you use the hand tool in adobe viewer, the grab point sticks with the mouse which is sort of intuitive. I'll check the example1.js.

**Comment by JeremyHeleine** [comment]
Yeah, I totally get it. For now, you will be able to only control the
movement speed, so actually it's not what you want.
I have an idea to implement the feature you want, so I'll do it as soon as
I can find some time.

**Comment by Kampii** [comment]
Hopefully! :) Let me know if I can help with a testing
